website/docs/general/blog-contributing-guide.md

```diff
@@ -0,0 +1,353 @@
+
+##### authorImageURL
+
+The author's GitHub avatar, it is related with `authorURL`. For the same reason described  above, this is an optional field. See below for how to get the avatar.
+
+##### keywords
+
+Required field, keyword, which is used to enhance better SEO performance. Usually the first three keywords are "APISIX", "Apache APISIX", "API Gateway", and the last two are the article's own keywords.
+
+##### description
+
+Required field, the description, which is used to enhance better SEO performance. Usually you can summarize the first or last paragraph of the article, forming about 120 words of text, and put it here.
+
+##### tags
+
+Required field, tag, which is used to categorize the blog. Each post has and can only have one tag. If more than one tag is suitable, please choose the most suitable one. The available tags and explanations are as follows. Please note that these tags and rules of applying tags could change over time.
+
+- **Community**: community related, e.g. "How to contribute to an open source project without writing code?"
+- **Events**: event-related, for example: online live stream, event previews, meetups, and online meeting, etc.
+- **Interview**: Interviews, e.g., Dr. Yang Li interview, Summer of Programming interview.
+- **Practical Case**: Best practices, easily confused with Technology. The factors that determine whether an article is a Technology or a Practical Case are: the content of the article and the subject of the description. For example, if the article is about "Running Apache APISIX on xxx platform", then it is a Practical Case; for example, if the article is about technical stuff, "Apache APISIX v.s. Envoy", then it is Technology.
+- **Release**: Release notes, this is better understood. It should be noted that the release notes inside the blog are polished, while the release notes inside the release are written by developers.
+- **Security**: Security vulnerability notification and methods to bypass security vulnerabilities, currently there are only two articles, very good to identify, generally have CVE-xxxxxxx is it.
+- **Technology**: Technical articles, easily confused with Practical Case. The factors that determine whether an article is a Technology or a Practical Case are: the content of the article and the subject of the description. For example, if the article is about "Running Apache APISIX on xxx platform", then it is a Practical Case; for example, if the article is about technical stuff, "Apache APISIX v.s. Envoy", then it is Technology.
+- **User Case**: User Case, this is also very straightforward. Please tell us how you use Apache APISIX and your feelings about Apache APISIX.
+
+#### How to get the authorImageURL
+
+1. Open Chrome
```

Comment:
How about clicking the avatar and copy URL directly? 😄

Reply:
![image](https://user-images.githubusercontent.com/2106987/143974668-5aa1c3e6-d5bf-495a-89cc-e8d6b9c61289.png)